src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link-logo"
+      >
+        <img
+          src="/img/logo-moyo.png"
+          alt="logo"
```

Comment:
The `alt` attribute should describe the image content. Instead of using a generic term like 'logo', provide a more descriptive text that conveys the content or purpose of the image.